app/Resources/views/themes/ezplatform/full/community_metrics.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends '@ezdesign/pagelayout.html.twig' %}
+
+{% block main %}
+    <main class="white-background">
+{% endblock %}
+
+{% block content %}
+    <div class="container article">
+        <h1>{{ ez_content_name(content) }}</h1>
+
+        {% if not ez_is_field_empty(content, 'intro') %}
+            <div class="article-intro">
+                {{ ez_render_field(content, 'intro') }}
+            </div>
+        {% endif %}
+
+        <div class="article-body">
+            {{ ez_render_field(content, 'body') }}
+        </div>
+
+        <div class="iframe-container">
```

Comment:
Could you please put the iframe outside the `container`? It will give more room to display all.